example/cmesh/t8_cmesh_geometry_examples.cxx

```diff
@@ -143,6 +143,26 @@ main (int argc, char **argv)
     t8_forest_unref (&forest);
   }
 
+  {
+    const char *prefix_cmesh = "t8_quadrangulated_spherical_surface_cmesh";
+    const char *prefix_forest = "t8_quadrangulated_spherical_surface_forest";
```

Comment:
Maybe you want to rename the squared disk to quadrangulated disk as well

Reply:
I'll do this in a separate PR.